.gitattributes

```diff
@@ -1,3 +1,4 @@
-Dockerfile* linguist-language=Dockerfile
+Dockerfile* linguist-language=Dockerfile eol=lf
 vendor.mod linguist-language=Go-Module
 vendor.sum linguist-language=Go-Checksums
+*.sh eol=lf
```

Comment:
Looks like you missed a newline at the end 😅

Reply:
Fixed. Thanks for catching this :)